packages/relay/src/lib/eth.ts

```diff
@@ -2012,54 +2012,52 @@ export class EthImpl implements Eth {
       return cachedResponse;
     }
 
-    const cacheKeySyntheticLog = `${constants.CACHE_KEY.SYNTHETIC_LOG_TRANSACTION_HASH}${hash}`;
-    const cachedLog = await this.cacheService.getSharedWithFallback(
-      cacheKeySyntheticLog,
-      EthImpl.ethGetTransactionReceipt,
-      requestIdPrefix,
-    );
+    const receiptResponse = await this.mirrorNodeClient.getContractResultWithRetry(hash, requestIdPrefix);
+    if (receiptResponse === null || receiptResponse.hash === undefined) {
+      // handle synthetic transactions
+      const syntheticLogs = await this.common.getLogsWithParams(
```

Comment:
So if I understand this correctly, the mirror node now supports using a transaction hash to fetch contract results in the logs, so the relay's existing calls to the mirror node didn't need to be updated.  The response just needed to be handled, correct?